Firestore/Swift/Source/PropertyWrapper/FirestoreQuery.swift

```diff
@@ -39,10 +48,54 @@ public enum DecodingFailureStrategy {
 ///       }
 ///     }
 ///
-/// In this example, `FirestoreQuery` will fetch all documents from the `fruits`
-/// collection, filtering only documents whose `isFavourite` attribute is equal
-/// to `true`, map members of result set to the `Fruit` type, and make them
-/// available via the wrapped value `fruits`.
+/// `FirestoreQuery` also supports returning a `Result` type. The `.success` case
+/// returns an array of elements, whereas the `.failiure` case returns an error
+/// in case mapping the Firestore docments wasn't successful:
+///
+///     struct ContentView: View {
+///       @FirestoreQuery(
+///         collectionPath: "fruits",
+///         predicates: [.whereField("isFavourite", isEqualTo: true)]
+///       ) var fruitResults: Result<[Fruit], Error>
+///
+///     var body: some View {
+///       if case let .success(fruits) = fruitResults {
+///         List(fruits) { fruit in
+///           Text(fruit.name)
+///         }
+///       } else if case let .failure(error) = fruitResults {
+///         Text("Couldn't map data: \(error.localizedDescription)")
+///       }
+///     }
+///
+/// Alternatively, the _projected value_ of the property wrapper provides access to
+/// the `error` as well. This allows you to display a list of all successfully mapped
+/// documents, as well as an error mesage with details about the documents that couldn't
+/// be mapped successfully (e.g. because of a field name mismatch).
+///
+///     struct ContentView: View {
+///       @FirestoreQuery(
+///         collectionPath: "mappingFailure",
+///         decodingFailureStrategy: .ignore
+///       ) private var fruits: [Fruit]
+///
+///       var body: some View {
+///         VStack(alignment: .leading) {
+///           List(fruits) { fruit in
+///             Text(fruit.name)
+///           }
+///           if $fruits.error != nil {
+///             HStack {
+///               Text("There was an error")
+///                 .foregroundColor(Color(UIColor.systemBackground))
+///               Spacer()
+///             }
+///             .padding(30)
+///             .background(Color.red)
+///           }
+///         }
+///       }
+///     }`
```

Comment:
nit: looks like there is a trailing ` character

Reply:
Eagle-eyed review! I wonder how this sneaked in...